modules/local/subworkflow/build_indices.nf

```diff
@@ -1,46 +1,47 @@
 /*
 ================================================================================
-                                BUILDING INDEXES
+                                BUILDING INDICES
 ================================================================================
 */
 
-// And then initialize channels based on params or indexes that were just built
+// And then initialize channels based on params or indices that were just built
 
 include { BUILD_INTERVALS }                            from '../process/build_intervals.nf'
 include { BWA_INDEX }                                  from '../../nf-core/software/bwa/index/main.nf'
 include { BWAMEM2_INDEX }                              from '../../nf-core/software/bwamem2_index.nf'
 include { CREATE_INTERVALS_BED }                       from '../process/create_intervals_bed.nf'
-include { GATK_CREATESEQUENCEDICTIONARY as GATK_DICT } from '../../nf-core/software/gatk_createsequencedictionary.nf'
+include { GATK_CREATESEQUENCEDICTIONARY as GATK_DICT } from '../../nf-core/software/gatk/createsequencedictionary.nf'
 include { HTSLIB_TABIX as TABIX_DBSNP;
           HTSLIB_TABIX as TABIX_GERMLINE_RESOURCE;
           HTSLIB_TABIX as TABIX_KNOWN_INDELS;
           HTSLIB_TABIX as TABIX_PON;}                  from '../../nf-core/software/htslib_tabix'
-include { SAMTOOLS_FAIDX }                             from '../../nf-core/software/samtools_faidx.nf'
+include { SAMTOOLS_FAIDX }                             from '../../nf-core/software/samtools/faidx.nf'
 
 workflow BUILD_INDICES{
     take:
-        dbsnp
-        fasta
-        germline_resource
-        known_indels
-        pon
-        step
-        tools
+        dbsnp             // channel: [optional]  dbsnp
+        fasta             // channel: [mandatory] fasta
+        germline_resource // channel: [optional]  germline_resource
+        known_indels      // channel: [optional]  known_indels
+        pon               // channel: [optional]  pon
+        step              //   value: [mandatory] starting step
+        tools             //    list: [optional]  tools to run
+        bwa_index_opts    //     map: options for BWA_INDEX module
 
```

Comment:
Hm, somehow it would feel more intuitive for me to have all the mandatory inputs first, and then the optional ones. Also is the bwa module optional or mandatory?

Reply:
That's a good point.
bwa_opts are actually mandatory, but hidden.
I think I began sorting them alphabetically, and at some point, did not follow through.
I can fix that.

Reply:
auto-sorting of parameters would be a cool feature to have :D I am reviewing now, maybe I will just add it there as a comment too, so everything is in one place